OpenSim/Sandbox/MatlabScripts/c3d_reading.m

```diff
@@ -80,28 +79,24 @@
         rotatedData = [rotationMatrix'*vectorData']';
```

Comment:
It's easy to define rotation matrices the wrong way around, and the three transposes here could be confusing. Perhaps apply the transpose to `rotationMatrix` when it's defined and include a comment on L57 indicating whether new data = old data after rotation by Rot about X, or the other way around. The last transpose can be removed because L82 accesses the elements individually anyway.